plugins/woocommerce/uninstall.php

```diff
@@ -33,8 +34,8 @@
  * and to ensure only the site owner can perform this action.
  */
 if ( defined( 'WC_REMOVE_ALL_DATA' ) && true === WC_REMOVE_ALL_DATA ) {
-	// Drop WC Admin tables.
-	include_once dirname( __FILE__ ) . '/includes/class-wc-install.php';
+	// Load WooCommerce so we can access the container, install routines, etc, during uninstall.
+	require_once __DIR__ . '/woocommerce.php';
```

Comment:
✍🏽 I'll just quickly note, requiring this file takes care of requiring `/includes/class-wc-install.php`, hence why I removed that existing line.